[rospy.client][INFO] 2020-02-02 23:14:33,794: init_node, name[/sr_determine_colors_7492_1580665473755], pid[7492]
[xmlrpc][INFO] 2020-02-02 23:14:33,795: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-02 23:14:33,795: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:41637/]
[rospy.init][INFO] 2020-02-02 23:14:33,796: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:41637/]
[rospy.impl.masterslave][INFO] 2020-02-02 23:14:33,796: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:41637/
[rospy.registration][INFO] 2020-02-02 23:14:33,798: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-02 23:14:33,801: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-02 23:14:33,896: registered with master
[rospy.rosout][INFO] 2020-02-02 23:14:33,897: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-02 23:14:33,899: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-02 23:14:33,901: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-02 23:14:33,913: topic[/usb_cam/image_rect_color] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:38957/], count 0
[rospy.internal][INFO] 2020-02-02 23:14:34,144: topic[/rosout] adding connection to [/rosout], count 0
[rosout][ERROR] 2020-02-02 23:14:34,184: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:34,420: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:34,652: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:34,882: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:35,118: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:35,348: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:35,564: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:35,800: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:36,030: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:36,900: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:37,125: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:37,986: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:38,254: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:38,539: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:38,795: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:39,080: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:39,308: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rosout][ERROR] 2020-02-02 23:14:39,560: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rospy.core][INFO] 2020-02-02 23:14:40,103: signal_shutdown [signal-2]
[rosout][ERROR] 2020-02-02 23:14:40,185: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7ff66dc39c20>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector1.py", line 81, in callback
    print("location:    " +str(self.yo.location))
AttributeError: sr_determine_colors instance has no attribute 'yo'

[rospy.internal][INFO] 2020-02-02 23:14:40,188: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-02 23:14:40,188: topic[/usb_cam/image_rect_color] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:38957/
[rospy.impl.masterslave][INFO] 2020-02-02 23:14:40,190: signal-2
[rospy.core][INFO] 2020-02-02 23:14:40,806: signal_shutdown [atexit]
